docs/reference/software-versions.mdx

```diff
@@ -14,6 +14,50 @@ Release candidates are software releases that are also released to the [Testnet]
 
 [testnet]: ./networks.mdx
 
+## Protocol 21: Preview 1 (Testnet only, April 12, 2024)
+
+### Software
+
+| Software | Version |
+| --- | --- |
+| XDR | `v21.0.1` |
```

Comment:
stellar-xdr also has a release https://github.com/stellar/stellar-xdr/releases/tag/v21.1. Should we add that and separate from rs-stellar-xdr?